### ISSUE TITLE
Upper limit of batch size in the image generation stage.

### ISSUE THREAD

**Issue by zgxiangyang** [jamesli1618/Obj-GAN#7]
When 4 "16G V100" GPUs are used, the batch size is only up to 16.
At the same time, the max batch size can only be set to 4, while only one GPU with 16G memory is available.
Is this normal?
I see the batch size can be set up to 64 in your paper.

**Comment by zgxiangyang** [comment]
And the training process was slow, It took about 6 hours to iterate 1 epoch for the case of  "4 X 16G V100" GPUs. So 60 epochs may will take 15 days.

**Comment by jamesli1618** [comment]
Thanks a lot for your interests. The speed and batch size is normal.
For speed issue, if you want to quickly evaluate a modification based on our model, a trick is to just do the evaluation after the first 2 or 3 epochs. The trend of the eval metrics is already very meaningful.
For batch size issue, you may want to try the sync normalization, which may improve the performance based on small batch size.

We will release the pretrained image generation model in one or two days.

**Comment by jamesli1618** [comment]
The pretrained image generation checkpoint is released.
